compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Pretty.hs

```diff
@@ -394,6 +406,8 @@ instance Pretty Update where
       pPrintAppKeyword lvl prec "ufetch_by_key" [tplArg retrieveByKeyTemplate, TmArg retrieveByKeyKey]
     ULookupByKey RetrieveByKey{..} ->
       pPrintAppKeyword lvl prec "ulookup_by_key" [tplArg retrieveByKeyTemplate, TmArg retrieveByKeyKey]
+    UTryCatch t e1 x e2 -> keyword_ "try" <-> pPrintTyArg lvl t <-> pPrintTmArg lvl e1
+      <-> keyword_ "catch" <-> pPrintPrec lvl precParam x <-> keyword_ "." <-> pPrintTmArg lvl e2
```

Comment:
We might a bit more verticality (is that a word?) here. But it's hard to tell if we don't have any producers for these AST nodes. Thus, let's have a look at the resulting output once we have the parser done. If we forget it, it's also not the end of the world since this is not for users anyway.

Reply:
👍 Makes sense.